lib/matplotlib/tests/test_triangulation.py

```diff
@@ -266,6 +266,15 @@ def test_tripcolor_color():
     ax.tripcolor(x, y, facecolors=[1, 2])  # faces
 
 
+def test_tripcolor_clim():
+    (a, b, c) = np.random.rand(10), np.random.rand(10), np.random.rand(10)
```

Comment:
We remove the randomness in tests by seeding the random number generator.
```suggestion
    np.random.seed(19680801)
    (a, b, c) = np.random.rand(10), np.random.rand(10), np.random.rand(10)
```
N.b. 19680801 is in honor of the original author of matplotlib, John Hunter. It's his birth date.

Reply:
Thank you! I'll make sure to remember that for next time.